packages/front-end/services/DefinitionsContext.tsx

```diff
@@ -21,9 +21,14 @@ import { ExperimentMetricInterface, isFactMetricId } from "shared/experiments";
 import { SavedGroupInterface } from "shared/src/types";
 import { MetricGroupInterface } from "back-end/types/metric-groups";
 import { CustomField } from "back-end/types/custom-fields";
+import {
+  ExperimentInterfaceStringDates,
+  ExperimentTemplateInterface,
+} from "back-end/types/experiment";
 import useApi from "@/hooks/useApi";
 import { useLocalStorage } from "@/hooks/useLocalStorage";
 import LoadingOverlay from "@/components/LoadingOverlay";
+import { useExperiments } from "@/hooks/useExperiments";
```

Comment:
Do not use this. Fetching all experiments is very expensive and should not be done like this on every page load in the app.